service/models.py

```diff
@@ -144,6 +147,12 @@ def deserialize_util(self, data):
             else:
                 raise TypeError
 
+        if data.get("active"):
+            if isinstance(data.get("active"), bool):
+                setattr(self, "active", data.get("active"))
+            else:
+                raise TypeError
+
```

Comment:
Try to rebase the branch over #68's branch, may get merge conflicts later.

Reply:
I am expecting merge conflicts. Lets see